### ISSUE TITLE
model: pytorch: Custom exception when loss function is misspelled

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#910]
Need helpful error messages when loss function is misspelled, try https://intel.github.io/dffml/master/tutorials/neuralnetwork.html with a misspelling of the loss function to see the current error.
